components/list/README.md

```diff
@@ -32,7 +32,6 @@ The `d2l-list` is the container to create a styled list of items using `d2l-list
 - `getListItemCount`: gets the length of the items within the list
```

Comment:
There's a Future Enhancement at the bottom of this page that talks about header that can now be removed. 🎉